src/repositories/ColorPaletteRepository.ts

```diff
@@ -65,7 +65,7 @@ export function mapColorPaletteApiToFrontifyColorPalette(colorPaletteApi: ColorP
 }
 
 export function mapFrontifyColorPaletteCreateToColorPaletteApiCreate(
```

Comment:
Should update method name as well:
```suggestion
export function mapColorPaletteCreateToColorPaletteApiCreate(
```